src/MasonryLayout/MasonryLayout.tsx

```diff
@@ -193,12 +193,16 @@ export default ({
         };
 
         const action = () => {
-            const noOfColumns = Math.floor(
-                current.clientWidth /
-                    (minWidth + Number.parseInt(styles.margin, 10))
-            );
+            const noOfColumns =
```

Comment:
Is this variable `numberOfColumns`, in that case, should it not be called `numberOfColumns`?